VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-list.component.ts

```diff
@@ -98,6 +101,7 @@ export class JudgeHearingListComponent implements OnInit, OnDestroy {
 
     onConferenceSelected(conference: ConferenceForHostResponse) {
         this.logger.debug('[JudgeHearingList] - Signing into judge waiting room', { conference: conference.id });
+        this.hearingVenueFlagsService.setHearingVenueIsScottish(conference.hearing_venue_is_scottish);
```

Comment:
Similar to above maybe we can have a route guard on the paths related to conferences that would handle this rather than having it in multiple places.

Reply:
not sure if I am following this, do you mean we have a route guard that we get conference details from?

Reply:
but as I have mentioned in my comment above, the change I have made is following the A/C and there is possibility that there might be other edge cases we might need to address however it will fall out of the scope of this ticket

Reply:
That's fine. I think that could be a more robust solution but given the ACs I'm happy to go ahead with this as is.